src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskSearchControllerTest.java

```diff
@@ -325,51 +325,6 @@ public void should_return_200_with_tasks_sorted_on_next_hearing_date_desc() {
             .forEach(task -> common.cleanUpTask(task.getTaskId()));
     }
 
-    @Test
-    public void should_return_200_with_tasks_sorted_on_next_hearing_date_asc() {
```

Comment:
this is checking we can sort nextHearingDate asc. Is this covered ini IT?

Reply:
No. Need to add into CftQueryServiceITTest::sortByFieldScenario

Reply:
Re-introduce